lib/api/model/model.dart

```diff
@@ -1,5 +1,7 @@
 import 'package:json_annotation/json_annotation.dart';
 
+import '../../model/algorithms.dart';
+import '../route/messages.dart';
```

Comment:
```suggestion
```

This is a wrong-direction import: specific routes should import from the model types, but not vice versa.

I think this was here only for a reference in dartdoc. It's acceptable to have those links not work when the alternative is a wrong-way import.